validation-test/Interop/Cxx/deserialize-debug.swift

```diff
@@ -0,0 +1,23 @@
+// RUN: %target-swift-frontend -enable-cxx-interop -I %S/Inputs %s -emit-ir -g | %FileCheck %s
```

Comment:
Could you move this test to `test/Interop/Cxx`? We would morph it into actual debug info test in future when the bug is fixed.